cmd/nodeUtil.go

```diff
@@ -88,6 +88,8 @@ func (cluster *Cluster) CreateNodes(suffix string, template Node, datacenters []
 }
 
 func (cluster *Cluster) ProvisionNodes(nodes []Node) error {
+	nodes = cluster.Nodes
```

Comment:
this is not the right way, as the only time we need to provision all nodes, it's during creation. When adding new workers, this statement will cause unintended side-effects.